dev/io.openliberty.microprofile.openapi.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -20,7 +20,7 @@ mpOpenAPI=MicroProfile OpenAPI
 mpOpenAPI.desc=Configuration for MicroProfile OpenAPI.
 
 docPath=MicroProfile OpenAPI document endpoint path
-docPath.desc=Path to obtain OpenAPI 3.0/Swagger documents.
+docPath.desc=Specifies the path for obtaining OpenAPI documents. If no value is specified, the path is set to /openapi
 
 uiPath=MicroProfile OpenAPI UI endpoint path
-uiPath.desc=Path for the OpenAPI UI.
\ No newline at end of file
+uiPath.desc=Specifies the path for accessing the MicroProfile OpenAPI UI. If no value is specified, the path is set to $docPath/ui
```

Comment:
Should have a period at the end of the sentence: If no value is specified, the path is set to $docPath/ui.

Reply:
Periods have now been added